pkg/apis/stork/v1alpha1/migration.go

```diff
@@ -21,6 +21,7 @@ type MigrationSpec struct {
 	StartApplications            *bool             `json:"startApplications"`
 	PurgeDeletedResources        *bool             `json:"purgeDeletedResources"`
 	SkipServiceUpdate            *bool             `json:"skipServiceUpdate"`
+	SkipNetworkPolicyCheck       *bool             `json:"skipNetworkPolicyCheck"`
```

Comment:
Need to add this field in MigrationSchedule as well.

Reply:
migrationschedule points to `migration` spec, there is no need to add it separately  in migrationschedule